frank/radial_fitters.py

```diff
@@ -742,6 +750,14 @@ def fit(self, u, v, V, weights=1):
 
             count += 1
 
+        if count < self._max_iter:
+            logging.info('\n    Convergence criterion met at iteration'
+                         ' {}'.format(count-1))
+        else:
+            logging.info('\n    Convergence criterion not met; fit stopped at'
+                         ' max_iter specified in your parameter file,'
+                         ' {}'.format(self._max_iter))
+
```

Comment:
Perhaps we should raise an exception, rather than logging a warning?
```suggestion
        if count >= self._max_iter:
            raise ValueError("Reached Maximum number of iterations before converging")
```

Reply:
I see your point, though in general I except fits may often reach the maximum number of iterations for datasets with properties we haven't seen yet.  I think if we want to raise an error because the fit hasn't converged:  
- we should be confident that the convergence criterion is robust 
- if we're not confident in it, we should increase the default number of iterations to some very high number. But I could see this potentially resulting in fits running much longer than needed (the convergence criterion has only been tested across a few datasets, not a wide diversity of them).

Reply:
Raising an error is the correct default behaviour. If you want to add an option to suppress it, feel free.